statistics/handle/internal/cache/lru/lru_cache.go

```diff
@@ -12,26 +12,29 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package handle
+package lru
 
 import (
 	"container/list"
 	"math"
 	"sync"
 
 	"github.com/pingcap/tidb/statistics"
-	handle_metrics "github.com/pingcap/tidb/statistics/handle/metrics"
+	"github.com/pingcap/tidb/statistics/handle/internal/cache"
+	"github.com/pingcap/tidb/statistics/handle/metrics"
 )
 
-type statsInnerCache struct {
+// StatsInnerCache is the LRU cache for statistics.
+type StatsInnerCache struct {
```

Comment:
Better to not expose the structure directly `StatsInnerCache` --> `statsInnerCache`. All outside users should use the corresponding interface instead of the `*StatsInnerCache`.

Reply:
It is in the internal. so It is just open to handle the package.